app/models/form/subsection.rb

```diff
@@ -30,7 +30,7 @@ def status(log)
 
     qs = applicable_questions(log)
     qs_optional_removed = qs.reject { |q| log.optional_fields.include?(q.id) }
-    return :not_started if qs.count.positive? && qs.all? { |question| log[question.id].blank? || question.read_only? || question.derived? }
+    return :not_started if qs.count.positive? && qs.all? { |question| !question.completed?(log) || question.read_only? || question.derived? }
```

Comment:
this means the status tag behaviour works correctly for the subsection, as the question id is `buyers_organisations` but actually we want to know if any of the 4 `preg...` ids (one for each checkbox) are yes in the log.